### PR TITLE
Add function to log vsb entire status

### PR DESCRIPTION
**Pull request by sbahar619**
I added function to log the entire status of the VolumeSnapshotBackup CR.

### CODE REVIEW

File: src/oadp_resources/oadp/datamover/volume_snapshot_backup.py [hunk 1]
```diff
@@ -117,3 +119,7 @@ def wait_for_done(self, wait_timeout=240, sleep=5):
             sleep=sleep,
             wait_timeout=wait_timeout
         )
+
+    def log_status(self):
```
Comment (amastbau) [inline src/oadp_resources/oadp/datamover/volume_snapshot_backup.py:123]: Please move to utils,
it is relevant not only for this resource but all.

File: src/oadp_resources/oadp/datamover/volume_snapshot_backup.py [hunk 2]
```diff
@@ -53,11 +53,13 @@ def replication_source_completed(self):
             conditions[1].type == "Synchronizing" and \
             not conditions[1].status
 
-    def done(self):
+    def done(self, log_status=False):
```
Comment (amastbau) [inline src/oadp_resources/oadp/datamover/volume_snapshot_backup.py:56]: do we really need to flag this?

Comment (PrasadJoshi12) [reply src/oadp_resources/oadp/datamover/volume_snapshot_backup.py:56]: +1 we should log by default

Comment (sbahar619) [reply src/oadp_resources/oadp/datamover/volume_snapshot_backup.py:56]: Do you think we should log all the status only for this done function by default?
Or for all the other phase functions?

File: src/oadp_resources/oadp/datamover/volume_snapshot_backup.py (hunk 1 above)
Comment (sbahar619) [reply src/oadp_resources/oadp/datamover/volume_snapshot_backup.py:123]: True, thanks, I will patch it.

Comment (sbahar619) [reply src/oadp_resources/oadp/datamover/volume_snapshot_backup.py]: Done.

File: src/oadp_resources/oadp/datamover/volume_snapshot_backup.py (hunk 2 above)
Comment (sbahar619) [reply src/oadp_resources/oadp/datamover/volume_snapshot_backup.py]: I patched it , done.

**Pull request merged**